### DISCUSSION TITLE
PR review status not updating

### DISCUSSION THREAD

**Discussion by leticiawebb** [community/community#35372] [Pull Requests]
We have several repositories that require pull requests. When a user requests a review, the reviewer is able to perform the review and approve the request. The approval shows up on the discussion thread with a green checkmark. This is the case for all repos I have checked. 

When I go to the Pull Requests section for one specific repo and look for review:approved, no records are returned even though I can see the approval note and green check in pull requests for this repo. Every other repo returns correct results when I use this filter. I reviewed specific pull requests for multiple repositories to see if there were differences between the processes used between the repositories and was not able to find any significant differences.

- Pull requests are initiated by the developer.
- Commits are added to the pull request.
- The developer requests a review.
- Changes may or may not be requested.
- The pull request is approved.
- The commits are then merged.

The only difference is in most repositories "* Approved" shows up at the end of the line in the closed pull request list. This does NOT appear for this one specific repository.

What additional information would be helpful to include, or any thoughts on what else to check to see what is causing this behavior?

**Comment by Kve-sudo** [comment]
okay.  but this has to do with what I am looking to install pycharm and
python
